vue-components/stories/TextArea.stories.ts

```diff
@@ -107,6 +140,12 @@ export function all(): Component {
                 <div style="max-width: 95%; margin-top: 1em;">
                     <TextArea label="Loading" placeholder="Placeholder" :loading="true" value="When the text area is set to loading, an indeterminate progress bar will appear." />
                 </div>
+                <div style="max-width: 95%; margin-top: 1em;">
+                    <TextArea label="Warning" :error="{ type: 'warning', message: 'Warnings should appear underneath the text area, as long as they are passed to the component.' }" value="To set a warning message for the text area, pass an object with the type: 'warning' along with a message to the 'error' prop." />
+                </div>
+                <div style="max-width: 95%; margin-top: 1em;">
+                    <TextArea label="Warning" :error="{ type: 'error', message: 'Errors should appear underneath the text area, as long as they are passed to the component.' }" value="To set an error message for the text area, pass an object with the type: 'error' along with a message to the 'error' prop." />
```

Comment:
```suggestion
                    <TextArea label="Error" :error="{ type: 'error', message: 'Errors should appear underneath the text area, as long as they are passed to the component.' }" value="To set an error message for the text area, pass an object with the type: 'error' along with a message to the 'error' prop." />
```

Reply:
```suggestion
                    <TextArea label="Warning" placeholder="Placeholder" :error="{ type: 'error', message: 'Errors should appear underneath the text area, as long as they are passed to the component.' }" value="To set an error message for the text area, pass an object with the type: 'error' along with a message to the 'error' prop." />
```

Reply:
Itsy-bitsy correction. Everything else looks 💯 .

Reply:
OH! Thank you!